cmake/package.cmake

```diff
@@ -0,0 +1,74 @@
+# This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+#
+# This file is free software; as a special exception the author gives
+# unlimited permission to copy and/or distribute it, with or without
+# modifications, as long as this notice is preserved.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+# Define install candidates for SQL files
+include(GNUInstallDirs)
+install(DIRECTORY "${CMAKE_SOURCE_DIR}/sql" DESTINATION "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}")
+
+# Download the full database
+set(FULL_DATABASE_URL "https://github.com/TrinityCore/TrinityCore/releases/download/TDB335.21041/TDB_full_world_335.21041_2021_04_16.7z"
```

Comment:
This should be updated in https://github.com/jackpoz/TrinityCoreCron/tree/3.3.5-TDB

Reply:
I've made a PR here: https://github.com/jackpoz/TrinityCoreCron/pull/5